=======================
Loading Anaconda Module...
"data":    {'debug': False, 'batch_size': 4, 'source': 'CNET', 'num_workers': 5, 'train_dir': '/datasets/SemEval/semeval-2023-task-1-V-WSD-train-v1/train_v1', 'trial_dir': '/datasets/SemEval/semeval-2023-task-1-V-WSD-train-v1/trial_v1'}
"model":   {'load': False, 'name': 'CLIPFinetune'}
"output":  clip_ft_cnet_2.out
"seed":    12345
"trainer": {'accelerator': 'gpu', 'devices': 1, 'max_epochs': 10, 'precision': 16, 'gradient_clip_val': 1.0, 'val_check_interval': 1.0, 'resume_from_checkpoint': None, 'accumulate_grad_batches': 1, 'num_sanity_val_steps': 0}
{'load': False, 'name': 'CLIPFinetune'}

Data config {'debug': False, 'batch_size': 4, 'source': 'CNET', 'num_workers': 5, 'train_dir': '/datasets/SemEval/semeval-2023-task-1-V-WSD-train-v1/train_v1', 'trial_dir': '/datasets/SemEval/semeval-2023-task-1-V-WSD-train-v1/trial_v1'} 

/scratch/users/oince22/vwsd/scripts_google
Error executing job with overrides: []
Traceback (most recent call last):
  File "/scratch/users/oince22/vwsd/scripts_google/clip_ft_pred.py", line 35, in main
    train_dm.setup(stage='fit')
  File "/kuacc/users/oince22/.conda/envs/vwsd/lib/python3.10/site-packages/vwsd-0.0.1-py3.10.egg/vwsd/data.py", line 214, in setup
    self.data = self.load_split(split='train')
  File "/kuacc/users/oince22/.conda/envs/vwsd/lib/python3.10/site-packages/vwsd-0.0.1-py3.10.egg/vwsd/data.py", line 228, in load_split
    return VWSDDataset(
  File "/kuacc/users/oince22/.conda/envs/vwsd/lib/python3.10/site-packages/vwsd-0.0.1-py3.10.egg/vwsd/data.py", line 33, in __init__
    self._load_data()
  File "/kuacc/users/oince22/.conda/envs/vwsd/lib/python3.10/site-packages/vwsd-0.0.1-py3.10.egg/vwsd/data.py", line 59, in _load_data
    prompt = self._clean_scraped_data(scraped_item, complete_context, source=self.source)
TypeError: VWSDDataset._clean_scraped_data() got an unexpected keyword argument 'source'

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
